ec2/InstanceType.dhall

```diff
@@ -314,153 +315,377 @@ let alternatives =
             → λ(instanceType : InstanceType)
             → merge
                 { c5-large =
-                      [ InstanceType.c5d-large, InstanceType.c5n-large ]
-                    : List InstanceType
+                    let cross-instances =
+                          [ InstanceType.m4-large
```

Comment:
`m4-large` has "moderate" network performance compared to "up to 10 Gbps" on a `c5-large`. If you really want to list `m4-large` here I would consider hiding it behind the `permitWorseNetwork` flag

Reply:
It is also considered a `permitLowerClassCPU`, how you think it should be considered?

Reply:
only permit if both `permitLowerClassCPU` and `permitWorseNetwork` are `True`